lib/inherited_class_var.rb

```diff
@@ -60,7 +62,7 @@ def hidden_variable_name(variable_name)
     # @return [Array<Module>] inherited_ancestors of included_module (including self)
     def inherited_ancestors(included_module=InheritedClassVar)
       included_model_index = ancestors.index(included_module)
-      included_model_index == 0 ? [included_module] : ancestors[0..(included_model_index - 1)]
+      included_model_index == 0 ? [included_module] : (ancestors[0..(included_model_index - 1)] - [InvalidOptions])
```

Comment:
Here I remove specifically this one so I guess should be safe, for my was confusing to have : 

```
expect(ClassWithFamily.send(:inherited_ancestors)).to eql [ClassWithFamily, Parent, InvalidOptions]
```

Because `InvalidOptions` isn't an inherited ancestors class of `ClassWithFamily`